src/ImportDefinitionsBundle/Command/CreateExportCommand.php

```diff
@@ -0,0 +1,130 @@
+<?php
+/**
+ * Import Definitions.
+ *
+ * LICENSE
+ *
+ * This source file is subject to the GNU General Public License version 3 (GPLv3)
+ * For the full copyright and license information, please view the LICENSE.md and gpl-3.0.txt
+ * files that are distributed with this source code.
+ *
+ * @copyright  Copyright (c) 2016-2018 w-vision AG (https://www.w-vision.ch)
+ * @license    https://github.com/w-vision/ImportDefinitions/blob/master/gpl-3.0.txt GNU General Public License version 3 (GPLv3)
+ */
+
+namespace ImportDefinitionsBundle\Command;
+
+use CoreShop\Bundle\ResourceBundle\Controller\ResourceFormFactoryInterface;
+use CoreShop\Component\Resource\Metadata\MetadataInterface;
+use CoreShop\Component\Resource\Repository\RepositoryInterface;
+use Doctrine\Common\Persistence\ObjectManager;
+use ImportDefinitionsBundle\Model\ExportDefinition;
+use Pimcore\Console\AbstractCommand;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+final class CreateExportCommand extends AbstractCommand
```

Comment:
rename to `ImportExportDefinitionCommand`

Reply:
I think I need 2 commands because of different dependencies

Reply:
Yes, but CreateExportCommand isn't the best name IMO. I would name them `ImportExportDefinitionCommand` and `ImportImportDefinitionCommand` (odd name though, but CreateExportCommand isn't good, cause actually you create a Definition)

Reply:
How should be the name of the commands `import-definitions:import` is already used. That's why I switched to create. I think adding Definition to the class name is good 👍

Reply:
`import-definition:import-definition` or `export-definition:import-definition` looks a bit stupid, but I think thats the best.